### DISCUSSION TITLE
Suitable first issue?

### DISCUSSION THREAD

**Discussion by vcrn** [ClementTsang/bottom#929] [General]
Hi! I'd like to look into if I can contribute to bottom. Are there any good first issues that anyone can think of?

**Comment by ClementTsang** [comment]
I'm not totally sure off the top of my head, but I can get around to labeling some issues as suitable this weekend.

**Reply by vcrn** [reply]
Great, thanks!

**Reply by ClementTsang** [reply]
Sorry for taking so long, went through some issues and marked a few that don't seem to be too hard to do for a first-time contributor. Feel free to ask any questions.

**Reply by vcrn** [reply]
No worries, and thank you for taking the time!
